src/plugins/intel_gpu/tests/test_cases/generate_proposals_gpu_test.cpp

```diff
@@ -289,6 +290,11 @@ struct generate_proposals_test
         const auto rois_num_type = type_to_data_type<ROIS_NUM_T>::value;
 
         auto& engine = get_test_engine();
+        std::shared_ptr<cldnn::stream> stream = get_test_stream_ptr();;
+        if (engine.get_device_info().supports_immad) {
+            // Expected to test out-of-order queue-type
+            return;
+        }
```

Comment:
why is it disabled for dgpu? I cannot find lines for out_of_order queue...

Reply:
It was previously defined by get_test_stream_ptr(). But now there is an exception logc in get_test_stream_str().
This should be removed. 
Applied thanks.